packages/truffle-db/src/workspace/test/utils.ts

```diff
@@ -33,6 +39,18 @@ export class WorkspaceClient {
     );
     return result.data;
   }
+
+  async executePersisted (request, variables = {}) {
```

Comment:
Maybe add an optional parameter to `execute` instead of making a separate method